Source/WebCore/rendering/ReferencedSVGResources.h

```diff
@@ -57,13 +60,19 @@ class ReferencedSVGResources {
     static Vector<std::pair<AtomString, QualifiedName>> referencedSVGResourceIDs(const RenderStyle&);
     void updateReferencedResources(TreeScope&, const Vector<std::pair<AtomString, QualifiedName>>&);
 
-    // Clipping needs a renderer, filters use an element.
+    // Legacy: Clipping needs a renderer, filters use an element.
     static LegacyRenderSVGResourceClipper* referencedClipperRenderer(TreeScope&, const ReferencePathOperation&);
-    static RefPtr<SVGClipPathElement> referencedClipPathElement(TreeScope&, const ReferencePathOperation&);
     static RefPtr<SVGFilterElement> referencedFilterElement(TreeScope&, const ReferenceFilterOperation&);
 
     static LegacyRenderSVGResourceContainer* referencedRenderResource(TreeScope&, const AtomString& fragment);
 
+#if ENABLE(LAYER_BASED_SVG_ENGINE)
+    // LBSE: All element based.
+    static RefPtr<SVGClipPathElement> referencedClipPathElement(TreeScope&, const ReferencePathOperation&);
+    static RefPtr<SVGMarkerElement> referencedMarkerElement(TreeScope&, const String&);
```

Comment:
Is it needed to handle markers in this PR? Will tests regress if not?

Reply:
Nope, we can skip that part of the patch.

Reply:
Thanks!

Reply:
I was confused, we need those changes for RenderSVGPath.